src/main/java/Duke.java

```diff
@@ -1,10 +1,32 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Duke\nWhat can I do for you?\n\n");
+
+        int taskCounter = 0;
+        String line;
+        String[] tasks = new String[100];
+        Scanner in = new Scanner(System.in);
+        line = in.nextLine();
+
+        while(line.equals("bye") == false)
+        {
+            if(line.equals("list")==true)
+            {
+                for(int j=0;j<taskCounter;++j)
+                {
+                    System.out.println("        "+ (j+1)+". "+tasks[j]);
+                }
+            }
+            else
+            {
+                tasks[taskCounter++]= line;
+                System.out.println("        added: " + line);
+            }
+            line = in.nextLine();
+        }
+
+        System.out.print("      Bye. Hope to see you again soon!");
```

Comment:
I like the logical structure of your code, it makes it easy to follow and identify the different parts. I think that your code is clear and readable.